### ISSUE TITLE
UDP multicast documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5128]
|<img alt="santagada's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| santagada reported|
|-|-|
|Trac ID|trac#5128|
|Type|defect|
|Created|2011-06-01 15:01:09Z|

on http://twistedmatrix.com/documents/current/core/howto/udp.html there is an unused line in the multicast examples:

from twisted.application.internet import MulticastServer

The source there comes from doc/core/howto/listings/udp from what I gathered.

Attachments:

* [multicast.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8e8/8e819c4cd8cfc122ef968ad1034c87221d89403f/ebdca90c3fd6d1f269e4c44cbd18940667fd853f.patch) (1376 bytes) - added by exarkun on 2011-06-05 16:43:58Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5128 5128
type__defect defect
reporter__santagada santagada
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1306940469000000 1306940469000000
changetime__1307296421000000 1307296421000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

Attached a fix for two listing files; assume a .misc news fragment.  Please review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [32037]) Remove unused imports from two of the multicast example listings

Author: exarkun
Reviewer: thijs
Fixes: [#5128](https://github.com/twisted/twisted/issues/5128)

`twisted.application.internet.MulticastServer` was imported by unused by the
multicast client and server examples, so remove the imports.
